spec/models/article_spec.rb

```diff
@@ -138,7 +140,7 @@
         end
         params = { category_slug: 'category_2' }
         records = portal_2.articles.search(params)
-        expect(records.count).to eq(25)
+        expect(records.count).to eq(articles.size)
```

Comment:
@tejaswinichile This spec is useless, we have removed the pagination logic from the article model.